google-cloud-speech/CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Release History
 
+### 1.0.0 / 2020-05-05
+
+This is a major update with significant new features, improved documentation, and a fair number of breaking changes.
+
+Among the highlights:
+
+* Separate client libraries are now provided for specific service versions.
+* A new configuration mechanism makes it easier to control parameters such as endpoint address, network timeouts, and retry.
+* A consistent method interface using keyword arguments for all fields, and supporting request proto objects.
+* Helper methods for generating resource paths are more accessible.
+* An expanded, standardized streaming interface.
+
+See the MIGRATING file in the documentation for more detailed information, and instructions for migrating from earlier versions.
```

Comment:
Can you link to MIGRATING file from here?

Reply:
done. Thanks! (The link won't be active until we actually publish the library and its documentation.)